FabricExample/android/settings.gradle

```diff
@@ -1,9 +1,13 @@
 rootProject.name = 'FabricExample'
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
-includeBuild('../node_modules/react-native-gradle-plugin')
 
 if (settings.hasProperty("newArchEnabled") && settings.newArchEnabled == "true") {
+    includeBuild('../node_modules/react-native-gradle-plugin') {
+        dependencySubstitution {
+            substitute(module("com.facebook.react:react")).using(project(":"))
```

Comment:
Basically it's the same piece of code as in [react-native-screens](https://github.com/software-mansion/react-native-screens/blob/7c6e14b45834cf009c7069d46a528cf792d2a15e/FabricExample/android/settings.gradle#L4-L8) but moved inside `if`.

Reply:
ok

Reply:
FabricExample is meant to be used only with Fabric enabled so that makes no difference at all here. Let's wait until RN teams publishes an official tutorial to see what's the recommended configuration.